internal/oem/oem.go

```diff
@@ -84,7 +85,15 @@ func init() {
 	})
 	configs.Register(Config{
 		name:     "openstack",
-		provider: noop.Creator{},
+		provider: configdrive.Creator{},
+		baseConfig: types.Config{
+			Systemd: types.Systemd{
+				Units: []types.SystemdUnit{{
+					Name:   "coreos-metadata-sshkeys@.service",
```

Comment:
I'm not sure we can assume that `coreos-metadata` will be able to figure out the data source without user intervention.